lightwood/api/types.py

```diff
@@ -174,6 +174,7 @@ class StatisticalAnalysis:
     df_target_stddev: Optional[float]
```

Comment:
I think for this it would be super helpful to add the meaning/purpose of `target_class_distribution` vs `assumed_target_class_weight` to avoid user confusion

Reply:
call distribution == number of instances of each class; and weights == assigned importance to each class (whether this is related to the weights or not doesnt matter)